spec/features/publish/courses/new_tda_course_spec.rb

```diff
@@ -101,6 +103,7 @@
 
     when_i_click_on_the_course_i_created
     then_i_do_not_see_the_change_links_for_study_mode_funding_type_and_visa_sponsorship_on_basic_details
+    and_i_do_not_see_the_degree_requirements_row_on_description_tab
```

Comment:
A couple of small suggestion:

 - Explicitly state that you are moving off of the basic details tab and onto the description tab.
 
 - Click on the description tab rather than navigating to the URL.


```
when_i_click_on_the_course_description_tab
then_i_do_not_see_the_degree_requirements_row
```

I'll need to add a test for course length on the description tab too.

Reply:
Good suggestions! Done